x/examples/outline-connectivity-app/go.mod

```diff
@@ -0,0 +1,40 @@
+module outline_sdk_connectivity_test
+
+go 1.20
+
+require (
+	github.com/Jigsaw-Code/outline-sdk v0.0.5-0.20230828133257-302b739dffcd
```

Comment:
I think the latest version is `v0.0.4`, I prefer to use a released version of SDK.